ext/page_log/palm/palm.c

```diff
@@ -279,7 +290,7 @@ palm_compute_delay_us(PALM *palm, uint64_t avg_delay_us)
  *     Add any artificial delay or simulated network error during an object transfer.
  */
 static int
-palm_delay(PALM *palm)
+palm_delay(PALM *palm, WT_SESSION *session)
```

Comment:
Why do we now need the session argument here?

Reply:
`session` is used in a hidden way by `PALM_VERBOSE_PRINT`.  I've made that use explicit in 6a0cced7e4